### ISSUE TITLE
Feature with respect to updating transactions

### ISSUE THREAD

**Issue by Villain45** [manuelalferez/flowinance#46]
It would better if there was a option to edit the transactions which are already saved.
Would like to enhance this feature

**Comment by manuelalferez** [comment]
Absolutely, that's a great idea and definitely a feature that must be included! I was planning to implement it, but I never got around to it. Would you like to implement the feature? Let me know if I can help in any way; happy to assist! 😊 @Villain45

**Comment by Villain45** [comment]
Absolutely, I would like to implement it

**Comment by manuelalferez** [comment]
@Villain45 wanted to run Supabase locally for the project, but he had difficulties creating the schema. The thing is, the whole schema is currently defined by the types, and in the past, I manually created the tables in Supabase. I know, it's not ideal. To save us time in the future, the best approach is to integrate [Prisma](https://www.prisma.io/) into the project, so all migrations can be easily performed with a single command
